xlators/cluster/afr/src/afr-self-heal-common.c

```diff
@@ -323,7 +324,7 @@ afr_gfid_split_brain_source(xlator_t *this, struct afr_reply *replies,
             break;
 
         case GF_SHD_OP_SBRAIN_HEAL_FROM_BRICK:
-            ret = dict_get_str_sizen(xdata, "child-name", &src_brick);
+            ret = dict_get_str_sizen(req, "child-name", &src_brick);
             if (ret) {
                 gf_msg(this->name, GF_LOG_ERROR, 0, AFR_MSG_SPLIT_BRAIN,
                        "Error getting the source "
```

Comment:
Use SERROR_GETTING_SRC_BRICK as it is done @338.

Reply:
This seems unrelated to this patch it looks like there are other occurrences of `gf_msg` that need changing and can be done in a separate patch. I wanted to take this fix in sooner because it fixes spurious failures blocking other patches. @karthik-us is that okay?

Reply:
If we have a lot of such messages which needs similar changes, then I am fine with taking that as a separate patch.

Reply:
Thanks @karthik-us .
@pranithk Please merge after approving.